src/styles.ts

```diff
@@ -51,9 +51,10 @@ export class StyleCompiler extends EventEmitter {
     return new Promise((resolve, reject) =>
       sass.render(_.assign({file: file}, config.styles.sass, {
         sourceMapEmbed: true,
-        sourceMapContents: true,
+        sourceMapContents: true,       
+		importer: (url, _ , done) => done({file: url.startsWith('~') ? url.substring(1) : url}),
```

Comment:
This fails on Travis because of tsc errors:
```
src/styles.ts(55,14): error TS7006: Parameter 'url' implicitly has an 'any' type.
src/styles.ts(55,19): error TS7006: Parameter '_' implicitly has an 'any' type.
src/styles.ts(55,23): error TS7006: Parameter 'done' implicitly has an 'any' type.
```
There are two issues here:
* Typing is missing. The function's signature w.r.t. `@types/node-sass` is:
```typescript
interface Importer {
    (url: string, prev: string, done: (data: { file: string; contents: string; }) => void): void;
}
```
You might fix this either using explicit typing on each parameter, or tell TS that the function you provide should be of type `Importer`, i.e. `<sass.Importer>((url, prev, done) => done({file: url.startsWith('~') ? url.substring(1) : url}))` or `((url, prev, done) => done({file: url.startsWith('~') ? url.substring(1) : url})) as sass.Importer`
* The typing mentions that there should be a `file` parameter on the provided object. Is this an error with typing or in your code? In case of the first option,  just add `as any` behind the object provided to `done`.